### ISSUE TITLE
Improve Documentation

### ISSUE THREAD

**Issue by evan-magnusson** [OpenSourcePolicyCenter/dynamic#86]
Because of the many changes to the code, especially the functions, I need to go through and make sure everything is clear and well documented.  I'll probably do this once everything is merged into the dynamic folder, so it only has to be done once.
Checklist of files to document:
- [x] demographics.py
- [x] firm_funcs.py
- [x] household_funcs.py
- [x] income_nopoly.py
- [x] income_polynomials.py
- [x] labor_data.py
- [x] misc_funcs.py
- [x] run_simulation.py
- [x] SS.py
- [x] tax_funcs.py
- [x] TPI.py
- [x] wealth_data.py

**Comment by evan-magnusson** [comment]
On second thought, I'll do this before merging, that way it will serve as a final sweep of the code to make sure everything is as efficient as possible and no old, unneeded code is lingering from past versions.

**Comment by evan-magnusson** [comment]
Finished, see PR #90 .
